src/frontend/admin/src/theme/JoanieThemeProvider.tsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+import { createTheme, ThemeOptions, ThemeProvider } from "@mui/material/styles";
+import * as React from "react";
+import { PropsWithChildren, useMemo } from "react";
+
+type Props = {};
+export function JoanieThemeProvider(props: PropsWithChildren<Props>) {
+  const themeOptions: ThemeOptions = useMemo(
+    () => ({
+      width: {
+        navigation: 280,
+      },
```

Comment:
I'll revert properties WDYT ?

```suggestion
      navigation: {
        width: 280,
      },
```

Reply:
Yes I had the opposite logic but yours is smarter